src/components/PriceBar/styles.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from "styled-components";
+import { PageItemWrapper as FooterItemWrapper } from "../helpers/ItemWrapper";
+
+export const Container = styled.div`
+  width: 100%;
+  background-color: white;
+  color: black;
+  position: relative;
+`;
+
+export const WrapperOuter = styled.div`
+  width: 30%;
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  // border: 1px solid purple;
```

Comment:
According to my personal experience, commented code should be deleted before creating a pull request if they don't have an actual meaning in the code.